operations/node_management/Replace_a_Compute_Blade.md

```diff
@@ -245,21 +262,6 @@ Replace an HPE Cray EX liquid-cooled compute blade.
     cray fas actions create CUSTOM_DEVICE_PARAMETERS.json
     ```
 
-1. Update the System Layout Service (SLS).
```

Comment:
This step is not required, nor is it event correct procedure. This step is not needed due to the pre-req `The Slingshot fabric must be configured with the desired topology`.